platform/src/main/java/org/stellar/anchor/platform/config/PropertyPaymentObserverConfig.java

```diff
@@ -4,21 +4,22 @@
 import org.springframework.validation.Errors;
 import org.springframework.validation.ValidationUtils;
 import org.springframework.validation.Validator;
-import org.stellar.anchor.config.CirclePaymentObserverConfig;
+import org.stellar.anchor.config.PaymentObserverConfig;
 
 @Data
-public class PropertyCirclePaymentObserverConfig implements CirclePaymentObserverConfig, Validator {
+public class PropertyPaymentObserverConfig implements PaymentObserverConfig, Validator {
   private boolean enabled = false;
   private String trackedWallet = "all";
+  private String circleUrl;
```

Comment:
👀  the `circleUrl` property doesn't seem to be used anywhere in the code. Are you missing something here?

Reply:
👍 Removed